src/main/java/com/google/flourbot/api/DriveCloudSheet.java

```diff
@@ -19,6 +19,7 @@
 public class DriveCloudSheet implements CloudSheet {
   private static final String VALUE_INPUT_OPTION = "USER_ENTERED";
   private static final String INSERT_DATA_OPTION = "INSERT_ROWS";
+  private static final String SHEET_NAME = "Sheet1"; // Stretch goal: allow user specified
```

Comment:
Does this automatically name the sheet they provided in the website to Sheet1?

Reply:
Naw, it just assumes they created the sheet and didn't rename it away from the default